### ISSUE TITLE
Remove pnputil.exe

### ISSUE THREAD

**Issue by ConsciousHacker** [LOLBAS-Project/LOLBAS#200]
pnputil.exe requires the driver to be signed, example output is below

```
C:\Windows\System32\pnputil.exe -i -a .\test.inf
Microsoft PnP Utility

Processing inf :            test.inf
Adding the driver package failed : The third-party INF does not contain digital signature information.


Total attempted:              1
Number successfully imported: 0
```

**Comment by ConsciousHacker** [comment]
1. The detail you provided about should be added to the entry for it to be a constructive contribution that the community benefits from. Please submit a pull request and we'll be happy to review it.
2. Please read the following: https://github.com/LOLBAS-Project/LOLBAS#criteria
3. If we consider "proxy execution" in combination with attacks used by signed drivers, it would change the scope of the LOLBAS project.

**Comment by wietze** [comment]
I agree that although this utility certainly could be of use to an attacker, the functionality appears to be intended, and as such this project is not the right fit for `pnputil`. Feel free to remove @ConsciousHacker

**Comment by ConsciousHacker** [comment]
@wietze Closed as discussed
